src/Orc.Controls/Controls/StepBar/Views/StepBarItem.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<catel:UserControl x:Class="Orc.Controls.StepBarItem"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:xamlbehaviors="http://schemas.microsoft.com/xaml/behaviors"
+             xmlns:catel="http://schemas.catelproject.com"
+             xmlns:orccontrols="clr-namespace:Orc.Controls"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="400">
+
+    <xamlbehaviors:Interaction.Triggers>
+        <xamlbehaviors:EventTrigger EventName="MouseDown">
+            <xamlbehaviors:InvokeCommandAction Command="{Binding Path=ViewModel.QuickNavigateToItem, RelativeSource={RelativeSource AncestorType={x:Type orccontrols:StepBar}}}" 
```

Comment:
This should not be a quick navigate to action. It should be linked to `Command` on IStepBarItem so the end-user can decide how to respond to this event.